### PR TITLE
Added cosmos "github.com/PTC-Global/dcos-sdk-go/cosmos/api/lib. And f…

### PR DESCRIPTION
**Pull request by fsniper**
…ixed framework deleting.

### CODE REVIEW

File: marathon/resource_marathon_app.go [hunk 1]
```diff
@@ -11,6 +11,7 @@ import (
 	"strconv"
 	"time"
 
+	cosmos "github.com/PTC-Global/dcos-sdk-go/cosmos/api/lib"
```
Comment (luisdavim) [inline marathon/resource_marathon_app.go]: shouldn't we first create a PR for squeakysimple/dcos-sdk-go and use that here?

Comment (fsniper) [reply marathon/resource_marathon_app.go]: I created that pr and It's actually merged. But I think we may need to wait for the licensing issue, before changing this to squeakysimple/dcos-sdk-go

Comment (luisdavim) [reply marathon/resource_marathon_app.go]: we can keep this temporarily until we get our changes in `squeakysimple/dcos-sdk-go`

**Review by luisdavim — APPROVED** [review]

File: marathon/resource_marathon_app.go (hunk 1 above)
Comment (fsniper) [reply marathon/resource_marathon_app.go]: We created a pr for adding MIT license to the original repo. I'm updating the pr

**Pull request merged**